proxy/src/http/project.rs

```diff
@@ -223,8 +224,18 @@ mod handler {
         let fake_fee: Balance = 100;
 
         let mut reg = registry.write().await;
+        let maybe_coco_id = match input.maybe_coco_id {
```

Comment:
You should be able to use `Option::map` to transform the `Some` value and return `None` if no value is present.